lua/neotest-playwright/discover.lua

```diff
@@ -177,7 +177,7 @@ ____exports.isTestFile = function(file_path)
     if not file_path then
         return false
     end
-    local endings = {".spec.ts", ".test.ts", ".spec.js", ".test.js"}
+    local endings = {".spec.ts", ".spec.tsx", ".test.ts", ".test.tsx", ".spec.js", ".spec.jsx", ".test.js", ".test.js"}
```

Comment:
`.test.js` is listed twice

```suggestion
    local endings = {".spec.ts", ".spec.tsx", ".test.ts", ".test.tsx", ".spec.js", ".spec.jsx", ".test.js", ".test.jsx"}
```